terraform/environments/data-platform/s3.tf

```diff
@@ -58,6 +58,246 @@ module "s3-bucket" { #tfsec:ignore:aws-s3-enable-versioning
   tags = local.tags
 }
 
+module "data_s3_bucket" { #tfsec:ignore:aws-s3-enable-versioning
+  source = "github.com/ministryofjustice/modernisation-platform-terraform-s3-bucket?ref=v7.0.0"
+
+  bucket_prefix      = "data-${local.environment}"
+  versioning_enabled = false
+  # Refer to the below section "Replication" before enabling replication
+  replication_enabled = false
+  bucket_policy       = [data.aws_iam_policy_document.data_s3_bucket_policy_document.json]
+  providers = {
+    # Here we use the default provider Region for replication. Destination buckets can be within the same Region as the
+    # source bucket. On the other hand, if you need to enable cross-region replication, please contact the Modernisation
+    # Platform team to add a new provider for the additional Region.
+    aws.bucket-replication = aws
+  }
+
+  lifecycle_rule = [
+    {
+      id      = "main"
+      enabled = "Enabled"
+      prefix  = ""
+
+      tags = {
+        rule      = "log"
+        autoclean = "true"
+      }
+
+      transition = [
+        {
+          days          = 90
+          storage_class = "STANDARD_IA"
+          }, {
+          days          = 365
+          storage_class = "GLACIER"
+        }
+      ]
+
+      expiration = {
+        days = 730
+      }
+
+      noncurrent_version_transition = [
+        {
+          days          = 90
+          storage_class = "STANDARD_IA"
+          }, {
+          days          = 365
+          storage_class = "GLACIER"
+        }
+      ]
+
+      noncurrent_version_expiration = {
+        days = 730
+      }
+    }
```

Comment:
Not necessary if versioning is `false`, I believe

Reply:
Good point, will remove 👍🏻